Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTMalware.yaml

```diff
@@ -18,7 +18,7 @@ relevantTechniques:
   - T0882
 query: |
   SecurityAlert
-  | where ProductName == "Azure Security Center for IoT"
+  | where ProviderName == "IoTSecurity"
   | where AlertName contains "Malware"
```

Comment:
We can also use has_any in this case to avoid multiple 'or' conditions. Similar to the approach that you have used in Unauthorized PLC changes (Microsoft Defender for IoT) rule

Reply:
done in some analytic rules with multiple string value @vakohl